t/t2260-job-list.t

```diff
@@ -24,6 +24,15 @@ wait_jobid_state() {
 	flux job list-ids --wait-state=$2 $1 > /dev/null
 }
 
+test_expect_success 'create helper job submission script' '
+	cat >sleepinf.sh <<-EOT &&
+	#!/bin/sh
+	echo "job started"
+	sleep inf
```

Comment:
Not sure if it is worth fixing here if there's no demonstrated problem, but in researching https://github.com/flux-framework/flux-core/issues/5382 I found that there can be a race if a signal is delivered to a script like this between the echo and the start of the sleep. In this case the signal was ignored and the test timed out.

See the fix in https://github.com/flux-framework/flux-core/commit/50bb62c42a4b6319ecbcdf64e6745ca3d1d96976 which uses awk and blocks on a FIFO instead of sleeping so there are no extra processes involved, thus avoiding the potential race.

Reply:
Hmmmm, for this case, I'm not sure its been seen.  I guess there'd be a hang if the signal was ignored?

But thinking about this ... does this race technically exist basically for every job that every user will ever run?  like.

```bash
#!/bin/bash
flux submit -N4 foo1.sh
flux submit -N4 foo2.sh
...
```

I dunno if this is generally solvable??

Reply:
Signal delivery is generally known to be unreliable. There's no fixing it. Only an issue here because the testsuite depends on a certain behavior. Since we're not seeing the hang in these tests, I'm fine leaving it as is until/if there's a real problem. I was just mentioning it as an fyi